README.md

```diff
@@ -3,3 +3,7 @@
 This repository will eventually house the source material for the documentation for the [OpenTracing](http://opentracing.io/documentation/) project.
 
 For the time being, that source material can be found in [this repository](https://github.com/opentracing/opentracing.io/tree/master/_docs).
+
+## License
+
+By contributing to OpenTracing documentation repository, you agree that your contributions will be licensed under its [Apache 2.0 License]
```

Comment:
Looks like the reference to the license isn't being rendered as a link:

![image](https://user-images.githubusercontent.com/13387/37657408-289090c8-2c4b-11e8-9fea-b02d6de9a41b.png)

Reply:
Okay, I will change it.